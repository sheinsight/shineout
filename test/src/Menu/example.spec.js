import exampleTest from '../../example'

describe('Menu[snapshot]', () => {
  exampleTest('Menu', {
    ignore: 'example-3-vertical',
  })
})
