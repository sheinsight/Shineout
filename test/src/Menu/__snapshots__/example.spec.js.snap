// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Menu[site/pages/components/Menu/example-1-base.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-1-link.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Google
        </a>
      </li>
      <li>
        <a>
          strackoverflow
        </a>
      </li>
      <li>
        <a>
          github
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-2-horizontal.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-4-disabled.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-5-selected.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-6-itemRender.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          <span>
            <i />
             
            Navigation One
          </span>
        </a>
      </li>
      <li>
        <a>
          <span>
            <i />
             
            Navigation Two
          </span>
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          <span>
            <i />
             
            Navigation Three
          </span>
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          <span>
            <i />
             
            Navigation Four
          </span>
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-7-click.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Snapshot test: Menu[site/pages/components/Menu/example-8-dark.js] 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a>
          Navigation One
        </a>
      </li>
      <li>
        <a>
          Navigation Two
        </a>
        <ul>
          <li>
            <a>
              Option 1
            </a>
          </li>
          <li>
            <a>
              Option 2
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Three
        </a>
        <ul>
          <li>
            <a>
              Option 3
            </a>
          </li>
          <li>
            <a>
              Option 4
            </a>
            <ul>
              <li>
                <a>
                  Optic 1
                </a>
              </li>
              <li>
                <a>
                  Optic 2
                </a>
              </li>
            </ul>
          </li>
        </ul>
      </li>
      <li>
        <a>
          Navigation Four
        </a>
      </li>
    </ul>
  </div>
</div>
`;
